src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -3477,7 +3484,9 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
               if (!sym.initialize.hasFlag(IS_ERROR)) {
                 newStats += typedStat(tree) // might add even more synthetics to the scope
                 tree.getAndRemoveAttachment[CaseApplyInheritAccess.type].foreach(_ =>
-                  runReporting.warning(tree.pos, "access modifiers for `apply` method are copied from the case class constructor", Scala3Migration, sym))
+                  runReporting.warning(tree.pos, "access modifiers for `apply` method were copied from the case class constructor under Scala 3", Scala3Migration, sym))
```

Comment:
Wasn't sure the extra attachment would be needed if you only get cross-features you asked for, but now I'm not sure what the warning policy is. I will continue reading...

Reply:
One of the two annotations is stale now, it's either warn or new semantics